code/modules/mob/living/living.dm

```diff
@@ -291,10 +291,14 @@
 
 /mob/proc/get_contents()
 
-/mob/living/proc/lay_down()
+/mob/living/proc/lay_down(var/force = FALSE)//force will override the cooldown
```

Comment:
Weirdly enough, disco balls already use the parameter, even though it's never defined. I'd imagine that's exactly what it was for.